src/map/packets/message_system.h

```diff
@@ -26,6 +26,53 @@
 
 #include "basic.h"
 
+// Found in ROM/27/76.dat
+// clang-format off
+enum MSGSYSTEM_ID : uint16
+{
+    // MSGSYSTEM_COULD_NOT_ENTER                  = 0,   // You could not enter the next area. (duplicate)
+    // MSGSYSTEM_COULD_NOT_ENTER                  = 1,   // You could not enter the next area. (duplicate)
+    MSGSYSTEM_COULD_NOT_ENTER                  = 2,   // You could not enter the next area.
+    // MSGSYSTEM_COULD_NOT_ENTER                  = 3,   // You could not enter the next area. (duplicate)
+    // MSGSYSTEM_COULD_NOT_ENTER                  = 4,   // You could not enter the next area. (duplicate)
+    MSGSYSTEM_COULD_NOT_ENTER_YOUR_ROOM        = 5,   // You could not enter your room.
```

Comment:
If this is documented elsewhere, then you don't need entries here saying there are duplicates. Should we be using the earliest seen entry if there are duplicates rather than one right in the middle?

Reply:
If you remove the duplicate comments, you remove the need for clang-format on/off